docs/local_development.rst

```diff
@@ -0,0 +1,172 @@
+
+.. code::
+
+  mkdir jupyter-cartoframes
+  cd jupyter-cartoframes
+  git clone git@github.com:CartoDB/cartoframes.git
+
+4. Create virtualenv
+
+.. code::
+  
+  python3 -m virtualenv <your env name>
+
+5. Install jupyter 
+
+.. code::
+
+  pipenv install jupyter
+
+6. Install cartoframes in dev mode using pipenv
+
+.. code::
+
+  pipenv install -e cartoframes
+
+7. Launch virtual environment
+
+.. code::
+
+  pipenv shell
+
+8. Activate the virtual environment
+
+.. code::
+
+  source env/bin/activate
+
+9. Launch jupyter notebook
+
+.. code::
+
+  jupyter notebook
+
+
+CARTO VL
+^^^^^^^^
+
+We're going to clone the repository. We can do it in the jupyter-cartoframes directory we've created to set up our local workspace.
+
+.. code::
+
+  cd jupyter-cartoframes
+  git clone git@github.com:CartoDB/carto-vl.git
+  cd carto-vl
+
+
+The local instalation of CARTO VL is explained `in this guide <https://github.com/CartoDB/carto-vl/blob/master/DEVELOPERS.md/>`__ 
+
+Airship
+^^^^^^^^
+
+Let's clone the Airship repository as well.
+
+.. code::
+
+  cd jupyter-cartoframes
+  git clone git@github.com:CartoDB/airship.git
+  cd airship
+
+
+The local instalation of Airship is explained `in this guide <https://github.com/CartoDB/airship/blob/master/DEVELOPERS.md/>`__ 
+
+Run all the projects
+^^^^^^^^^^^^^^^^^^^^
+
+We've to serve now all the projects as follows:
+
+.. code::
+
+  +---------------------------+---------------------------+
+  |/carto-vl                  |/carto-vl                  |
+  |$ npm run build:watch      | $ npm run serve           |
+  |                           |                           |
+  |                           |                           |
+  |                           |                           |
+  +---------------------------+---------------------------+
+  |/airship                   |/airship                   |
+  |$ npm run dev              | $ npm run serve           |
+  |                           |                           |
+  |                           |                           |
+  |                           |                           |
+  +---------------------------+---------------------------+
+  |/cartoframes               |                           |
+  |(env)$ jupyter notebook    |                           |
+  |                           |                           |
+  |                           |                           |
+  |                           |                           |
+  +---------------------------+---------------------------+
+
+Load JavaScript libraries locally
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In order to get Airship and CARTO VL libraries we're serving locally in CARTOFrames, we need to instantiate the map indicating the paths:
+
+.. code:: python
+
+  import cartoframes
+  from cartoframes.contrib import vector
+  from cartoframes import CartoContext
+
+  context = CartoContext(
+      base_url='https://username.carto.com/', 
+      api_key='default_public'
```

Comment:
Idea: Put one of our public accounts here (VL, for instance). I don't know if there's one for CARTOFrames

The more people can copy and paste with clear results, the better imho.

What do you think @andy-esch ?

Reply:
We have the cartoframes account (`cartoframes`/`default_public`) that has example datasets, etc. More on that here: https://cartoframes.readthedocs.io/en/stable/example_context.html It might be better to use that for consistency, but I think it's a good idea to also point out that the CARTO VL acct is a resource too since it has a wider variety of datasets and there are so many more code examples that people can replicate.